### ISSUE TITLE
[Request] Allow distribution outside of ml window

### ISSUE THREAD

**Issue by kcoulsy** [Alumian/CEPGP-Retail#37]
From what I understand (unless I missed something)

You can only distribute the items while you are a ML and have the loot window open.

What I would suggest is allow the ML to open distribution and add items wherever they are.

The reason for this, is with loot trading in raids, it would be nice for the ML to loot all the items and do distribution as we go. Rather than leaving them behind to sort it out. Would be nicer for raids who want to be efficient with time, less downtime etc.. 

Thanks

**Comment by kcoulsy** [comment]
Yeah sounds right.

It's so they can stick with the group while we've moved onto the next trash/boss or whatever. But still have the tool distribute properly.

**Comment by Alumian** [comment]
Okay cool, so you can more or less do that at the moment. The feature that is currently in place is you can manually award GP to a single player and make a note for that entry. Does that sound suitable for this purpose?

**Comment by kcoulsy** [comment]
That would work for now, but we'd need to check the players epgp values manually, rather than having that nice window which show's who needs/greeds them, and the values all together.

**Comment by kcoulsy** [comment]
Doesn't need to be a complicated Add item to Distrubtion system or anything

Perhaps something simple like a command /cepgp distribute [item]

Which would just open up this screen so you can see who replies.

![image](https://user-images.githubusercontent.com/25744222/65947588-83a98980-e430-11e9-85f5-862fad9c6f98.png)

**Comment by Alumian** [comment]
Ah right I see the problem. Unfortunately, the way that the addon identifies what item needs to be given is based on the slot number of the loot table. If you close the loot window and reopen the window, the slots change if an item has been taken/given out. Although this would not impact the feature you're asking about, it would void a measure I have implemented to ensure that the correct item is given to the correct player.
Essentially, if you close the loot window, the addon forces you to stop accepting acknowledgements for the item to ensure that the correct item is given out. Hence why it clears everything if you stop looting. Also the GP is only awarded when the addon has received confirmation that the item has successfully been given out. Due to the current process of the addon, this is not possible outside of the master loot environment.

**Comment by kcoulsy** [comment]
Yeah I feared that might have been the case tbh, but thought it was worth suggesting anyway :)

**Comment by Alumian** [comment]
Definitely was worth a suggestion and thanks for submitting this. Please don't let this deter you from submitting feature requests in future.

Thanks!
